serverless/admin_guide/serverless-ossm-setup.adoc

```diff
@@ -1,6 +1,4 @@
 :_content-type: ASSEMBLY
-include::_attributes/serverless-document-attributes.adoc[]
-include::_attributes/ossm-document-attributes.adoc[]
```

Comment:
This is going to cause your variables to break until Julie fixes the service mesh attribute files.

Reply:
ACK.